apps/native-component-list/src/screens/Location/BackgroundLocationMapScreen.tsx

```diff
@@ -334,6 +328,54 @@ function BackgroundLocationMapView() {
   );
 }
 
+const PermissionsModal = () => {
+  const [showPermissionsModal, setShowPermissionsModal] = React.useState(true);
+  const [permission] = usePermissions(Location.getBackgroundPermissionsAsync);
+
+  return (
+    <Modal
+      animationType="slide"
+      transparent={false}
+      visible={!permission && showPermissionsModal}
+      onRequestClose={() => {
+        setShowPermissionsModal(!showPermissionsModal);
+      }}>
+      <View style={{ flex: 1, justifyContent: 'space-around', alignItems: 'center' }}>
+        <View style={{ flex: 2, justifyContent: 'center', alignItems: 'center' }}>
+          <Text style={styles.modalHeader}>Background location access</Text>
+
+          <Text style={styles.modalText}>
+            In order to keep the MapView updated, Native Component List requires access to your
+            location even while the app is backgrounded. Otherwise, your location on the map will
+            only be updated while the app is foregrounded.
```

Comment:
We need the sentence, "This app collects location data to enable updating the MapView in the background even when the app is closed or not in use."